### DISCUSSION TITLE
How to use with <script setup>

### DISCUSSION THREAD

**Discussion by MaximSedov** [hrynko/vue-pdf-embed#43] [Q&A] [answered]
Greetings! I do it according to the manual, but I get an error. How can this problem be solved? 
Vue 3 , Vite
`TypeError: isFunction is not a function at defineComponent`

**Comment by hrynko** [comment, marked as answer]
Hi @MaximSedov,

I have not been able to reproduce this issue. I initialized an empty Vite project (`vite@2.9.9`, `vue@3.2.25`, `vue-pdf-embed@1.1.2`) and created the following component:

```vue
<script setup>
import VuePdfEmbed from 'vue-pdf-embed'
const pdfSource = 'data:application/pdf;base64,<BASE64_ENCODED_PDF>'
</script>

<template>
  <VuePdfEmbed :source="pdfSource" />
</template>
```

It renders with no errors. So I guess the problem is not VuePdfEmbed. Can you try to reproduce the error in a clean environment?

**Reply by hrynko** [reply]
Hope you resolved the issue, otherwise, please let me know if this really needs to be fixed in the library.
